txscript/opcode_test.go

```diff
@@ -141,9 +140,13 @@ func TestOpcodeDisasm(t *testing.T) {
 				expectedStr = "OP_NOP" + strconv.Itoa(int(val))
 			}
 
-		// OP_UNKNOWN#.
+			// OP_UNKNOWN#.
 		case opcodeVal >= 0xc1 && opcodeVal <= 0xf8 || opcodeVal == 0xfc:
 			expectedStr = "OP_UNKNOWN" + strconv.Itoa(int(opcodeVal))
+
+		// OP_INVALID.
+		case opcodeVal == 0xf9:
```

Comment:
There is no need to do this.  Instead, just replace the `0xfa` entry in the map with `OP_INVALID249`.